### PR TITLE
Visualiser!

### PR DESCRIPTION
**Pull request by 42willow**
This code isn't actually that bad to my standards...

To do:

- [x] Fix toggle switch
- [x] Performance (disable JS when element hidden)

### CODE REVIEW

**Comment by 42willow** [comment]
#3

**Comment by 42willow** [comment]
Oops still some things that need to be fixed up

**Comment by 42willow** [comment]
image

**Comment by 42willow** [comment]
Minor issue: the song name doesn't change if in same album (spotify, idk about the others)

**Comment by Ju1-js** [comment]
Oops

**Comment by 42willow** [comment]
https://github.com/rocksdanister/lively/issues/1745

File: Dynamic/js/LivelyProperties.js [hunk 1]
```diff
@@ -102,7 +104,36 @@ function livelyPropertyListener(name, val) {
...
+      visualiser.classList.toggle("hide", !val);
+      break;
+    case "show-when-not-playing":
+      visualiser.classList.toggle("show-when-not-playing", !val);
+      break;
+    case "sorting-mode":
+      switch (val) {
+        case 0:
+          sortSoundArray = concentratedSort;
+          numDots = 128;
```
Comment (Ju1-js) [inline Dynamic/js/LivelyProperties.js]: Why is numDots lower only for concentratedSort?

Comment (Ju1-js) [reply Dynamic/js/LivelyProperties.js]: Oh, the original was like that.

Comment (42willow) [reply Dynamic/js/LivelyProperties.js]: Yes I just copy pasted most of the code :)

Comment (42willow) [reply Dynamic/js/LivelyProperties.js]: I should probably include the license one sec...

**Review by Ju1-js — APPROVED** [review]
Looks good!

**Pull request merged**